test/score.test.js

```diff
@@ -0,0 +1,43 @@
+const Score = require('../lib/score')
+
+describe('Score class', () => {
+  beforeEach(() => {
+    score = new Score()
+  })
+
+  it('create an instance of Score', () => {
+    expect(score).toBeInstanceOf(Score)
+  })
+
+  it('initialize with a total of 0', () => {
+    expect(score.total).toEqual(0)
+  })
+
+  describe('addToTotal', () => {
+    it('sums the latest frame added', () => {
+      const gameDouble = { frames: [[1, 4]], isSpare: () => false, isStrike: () => false }
+      const score = new Score(gameDouble)
+      expect(score.addToTotal()).toEqual(5)
+    })
+
+    it('adds the bonus with the sum of last frame added', () => {
+      const gameDouble1 = { frames: [[6, 4], [5, 5]], isSpare: () => true, isStrike: () => false }
+      const score = new Score(gameDouble1)
+      expect(score.addToTotal()).toEqual(15)
+    })
+  })
+
+  describe('addBonus', () => {
+    it('returns a bonus if a spare is scored', () => {
+      const gameDouble = { frames: [[6, 4], [4, 1]], isSpare: () => true }
+      const score = new Score(gameDouble)
+      expect(score.addBonus(4, 1)).toEqual(4)
+    })
+
+    it('returns a bonus if a strike is scored', () => {
+      const gameDouble = { frames: [[10, 0], [5, 3]], isSpare: () => false, isStrike: () => true }
+      const score = new Score(gameDouble)
+      expect(score.addBonus(5, 3)).toEqual(8)
```

Comment:
A feature test (testing a full game) might be a good further test to add - I'm also not sure if the last frame will add as intended (not counting any strikes or spares on that frame for bonus points), just since I can't see anything for this